whipper/common/program.py

```diff
@@ -447,6 +458,10 @@ def getTagList(self, number, mbdiscid):
                 tags['MUSICBRAINZ_ALBUMID'] = mbidRelease
                 tags['MUSICBRAINZ_RELEASEGROUPID'] = mbidReleaseGroup
                 tags['MUSICBRAINZ_ALBUMARTISTID'] = mbidReleaseArtist
+                if classical and len(composers) > 0:
```

Comment:
No need to check `classical`… Non-classical works can have composers too, and if we are aware of composers, we may as well add them to the relevant tag.

Reply:
So i think the `--classical ` option should be removed just as you mentioned before.